src/monmove.cpp

```diff
@@ -683,7 +686,65 @@ void monster::move()
                 // Friendly fire and pushing are always bad choices - they take a lot of time
                 bad_choice = true;
             }
+            //Shove vehicles out of way
+            if( this->has_flag( MF_PUSH_VEH ) ) {
+                auto vp = g->m.veh_at( candidate );
+                if( vp ) {
+                    bool move_veh = false;
+                    bool rotate_veh = false;
+                    bool damage_veh = false;
```

Comment:
These flags are redundant: they're all either `true` or `false` simultaneously. Can be replaced with one.

Reply:
I've got rid of these flags.